Makefile

```diff
@@ -85,21 +85,21 @@ test: test-unit
 .PHONY: test-unit
 test-unit:
 	@echo Running unit tests
-	$(GO) test -mod=vendor -race -v -tags unit $(GO_PACKAGES)
+	$(GO) test -mod=vendor -timeout 30m -race -v -tags unit $(GO_PACKAGES)
```

Comment:
Is it worth extracting these flags into a a separate variable to DRY the code?

Reply:
Make sense, added in bd8425b